app/queries/get_embedded_content.rb

```diff
@@ -0,0 +1,39 @@
+module Queries
+  class GetEmbeddedContent
+    attr_reader(:target_content_id)
+
+    def initialize(target_content_id)
+      self.target_content_id = target_content_id
+    end
+
+    def call
+      if Document.find_by(content_id: target_content_id).nil?
+        message = "Could not find an edition to get embedded content for"
+        raise CommandError.new(code: 404, message:)
+      end
+
+      Presenters::Queries::EmbeddedContentPresenter.new(
+        target_content_id,
+        host_editions,
+      ).present
+    end
+
+  private
+
+    def host_editions
+      @host_editions ||= Edition.live
```

Comment:
Would we ever want people to be able to embed content in draft docs e.g. for previewing what's being published? Or are we starting with `live` for now?

Reply:
Embedding should work for draft documents so there is an argument that content block publishers might want to see ALL documents regardless of state.

[For our use case I've gone back to clarify what we need as our default case](https://gds.slack.com/archives/C062Z1FA2ES/p1724239210827159). 

In any case I can imagine that consumers might want embedded content in either state. I'll open this up to everything and then we can filter it down later 👍

Reply:
I've changed it to do draft and live, but not superseded or unpublished (though we don't have a factory for the latter).